File: challenges/2-tabuada/php/zuruckt/challenge.php
<?php

$num = $argv[1];

if (!is_numeric($num)) {
    echo 'Insira apenas valores numéricos!';
    die;
}

for ($i = 0; $i <= $num; $i++) {
    print('=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~='.PHP_EOL);
    printf('A tabuada do número %s é: '.PHP_EOL, $i);
    for ($j = 1; $j <= 10; $j++) {
        printf('%s * %s = %s'.PHP_EOL, $i, $j, $i * $j);
    }
}

File: challenges/2-tabuada/go/teocalvo/challenge.go
package main

import "fmt"

func tabuada(n int) {
	fmt.Println("===========================")
	fmt.Printf("A tabuada do número %d é:\n", n)
	for i := 1; i <= 10; i++ {
		fmt.Printf("  %d*%d = %d\n", n, i, n*i)
	}
}

func main() {
	var numInput int
	fmt.Scanf("%d", &numInput)
	for i := 1; i <= numInput; i++ {
		tabuada(i)
	}
}
